pkgs/development/python-modules/cvxpy/default.nix

```diff
@@ -33,6 +34,7 @@ buildPythonPackage rec {
     osqp
     scipy
     scs
+    cylp
```

Comment:
Is this an implicit dependency?

The build is fine on master. Generally optional dependencies are not included, as it increases minimal closure size.

Reply:
All of the solvers (osqp, scipy, scs, cvxopt included) here are optional.

Reply:
Coin-OR happens to be the _best_ open source solver that CVXPY supports as a backend, IMHO.

Reply:
meh, I guess this should only get used in domain specific  scenarios.  And the closure is already massive

```
$ nix path-info -Sh $(nix-build -A python3Packages.cvxpy)
/nix/store/ay0fkmz55x3327p8xnwgvra3qigd2rcm-python3.9-cvxpy-1.1.18	 406.4M
```